configuration/rules-dsl.md

```diff
@@ -206,6 +206,40 @@ For example, one z-wave device can be "zwave:device:c5155aa4:node14".
 > Note: You need to use quotes around `thingUID` if it contains special characters such as ':'.
 > The parser will not recognize the whole text correctly without quotes.
 
+{: #channel-based-triggers}
+### Channel-based Triggers
+
+You can listen for triggers from specific *trigger channels* that are provided by some addons.
+You can decide whether you want to catch only a specific trigger or any.
```

Comment:
I am not exactly sure what you mean by "or any".  So one can decide to catch a specific trigger, or one can decide to catch more than one trigger?  All triggers?  I have not used this, so I am not sure what you are getting at.

Reply:
That was just a dirty copy from the thing channel section above. 😄 

For my understanding "any" means, that all triggers that are available for this channel will fir the rule.

I will try to concretize that paragraph.

Reply:
Sounds correct to me.

Reply:
I will improve the corresponding thing-based-triggers sentence too then.